templates/Toolkit template/SoftwareName_Toolkit/SoftwareName_Adapter/CRUD/Update/_IUpdate.cs

```diff
@@ -25,24 +25,25 @@
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
+using BH.oM.Adapter;
 
 namespace BH.Adapter.$ext_safeprojectname$
 {
     public partial class $ext_safeprojectname$Adapter
     {
-        /***************************************************/
-        /**** Adapter overload method                   ****/
-        /***************************************************/
-
         // This method gets called when appropriate by the Push method contained in the base Adapter class.
-        // It gets called once per each Type, and if equal objects are found. The equality is tested through IEqualityComparer.
-        // IEqualityComparer must be implemented per each type. If not,
-        // by default the method first deletes the existing objects, then creates new ones.
-        protected override bool UpdateObjects<T>(IEnumerable<T> objects)
-        {
-            return base.UpdateObjects<T>(objects);
-        }
+		// Unlike the Create, Delete and Read, this method already exposes a simple implementation: it calls Delete and then Create.
+		// It can be overridden here keeping in mind the following:
+        // - it gets called once per each Type, and if equal objects are found;
+		// - the object equality is tested through this.AdapterComparers, that need to be implemented for each type.
+		// See the wiki for more info.
+		
+        // protected override bool IUpdate<T>(IEnumerable<T> objects, ActionConfig actionConfig = null) 
+        // {
+            // return base.UpdateObjects<T>(objects);
```

Comment:
Method does not exist if one un-comments it.
(suspect it should be base.IUpdate<T>...)